src/DynamoCoreWpf/Utilities/NodeContextMenuBuilder.cs

```diff
@@ -371,6 +372,8 @@ private static void DismissedAlertsMenuItemOnClick(object sender, RoutedEventArg
             if (!(e.OriginalSource is MenuItem menuItem)) return;
 
             NodeViewModel.ErrorBubble.UndismissMessageCommand.Execute(menuItem.Header);
+
+            Analytics.TrackEvent(Actions.Dismiss, Categories.NodeContextMenuOperations, "NodeAlerts");
```

Comment:
@zeusongit By looking at the code closely, I think the warning `Dismiss` location is around https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Preview/InfoBubbleView.xaml.cs#L1040 and this place is for `Undismiss`, let me know if you have question